phpBB/styles/prosilver/template/posting_body.html

```diff
@@ -1,4 +1,7 @@
 
+<!-- INCLUDEJS jquery.caret.js -->
```

Comment:
@paul999  How about using it for the following condition -
<!-- IF S_EDIT_DRAFT and not S_PRIVMSGS-->